pkg/controllers/scheduler/schedulingtriggers.go

```diff
@@ -49,7 +55,7 @@ Federated object changes:
 
 Propagation policy changes:
 1. policy creation
-2. generation change (spec update)
+2. semantics of policy change (see PropagationPolicySpec.ReschedulePolicy.Trigger.PolicyContentChanged for more detail)
```

Comment:
```suggestion
2. semantics of policy change, this can be toggled by the policy itself (see PropagationPolicySpec.ReschedulePolicy.Trigger.PolicyContentChanged for more detail)
```

Reply:
Let's also indicate in the comments that the below cluster changes can be toggled in the policy: `cluster labels`, `cluster taints` and `cluster apiresource`